automatic/waterfox/waterfox.nuspec

```diff
@@ -14,19 +14,16 @@
     <licenseUrl>https://github.com/MrAlex94/Waterfox/blob/master/LICENSE.md</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <projectSourceUrl>https://github.com/MrAlex94/Waterfox</projectSourceUrl>
-    <tags>waterfox admin web-browser firefox 64-bit</tags>
+    <tags>waterfox web-browser firefox 64-bit admin cross-platform foss</tags>
     <summary>Waterfox is basically a 64-bit version of Firefox.</summary>
     <description>
-## How and why Waterfox is here...
```

Comment:
IMHO this looks strange on the chocolatey.org page. 

Also there a lot of other packages which have differences to the about page of the project (eg. even here you already stripped the "About Waterfox" heading). 
But if you want to keep it I will revert this change.